src/libostree/ostree-core.h

```diff
@@ -158,6 +158,16 @@ typedef enum {
  * - a(s(taya{sv})) - Map of ref name -> (latest commit size, latest commit checksum, additional metadata), sorted by ref name
  * - a{sv} - Additional metadata, at the current time the following are defined:
  *   - key: "ostree.static-deltas", value: a{sv}, static delta name -> 32 bytes of checksum
+ *   - key: "ostree.summary.last-modified", value: t, timestamp (seconds since
+ *     the Unix epoch in UTC) when the summary was last regenerated (similar to
+ *     the HTTP `Last-Modified` header)
```

Comment:
Yeah, for the cases where the summary file is stored as a file somewhere (mtime is a bit too fragile, and is not signed), or where it’s propagated through several Avahi peers (similarly), or where a HTTP server is used but configuring `Last-Modified` is a pain to do (I don’t know how likely this is, but from experience, setting HTTP headers on static content is not always trivial).